### PR TITLE
update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kumiDa**
Resolves issue #1

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+ Contributing
```
Comment (sahitpj) [inline CONTRIBUTING.md]: Make this a heading using the # sign

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+* You can report bugs and request features using the [issues page][issues].
+
+[issues]: https://github.com/pydatagn/Community/issues
+
+* You can easily work on the code if you have a GitHub account ([get one][github]):
+
+[github]: https://github.com/join
+
+Fork the project, then clone the repo:
```
Comment (sahitpj) [inline CONTRIBUTING.md:20]: Can you make this more specific to our repositories

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+[pr]: https://github.com/pydatagn/Community/compare
+
+At this point you're waiting on us. We like to at least comment on pull requests
+as soon as possible. We may suggest some changes or improvements or alternatives.
+
+Some things that will increase the chance that your pull request is accepted:
+
+* Write a [good commit message][commit].
+
+[commit]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
```
Comment (sahitpj) [inline CONTRIBUTING.md]: I think we can add links to other repositories and a brief description about them, along with our website details.

**Review by sahitpj — COMMENTED** [review]
I was also thinking of adding a contributors hall of fame kinda thing, where we can maintain a list of people who have contributed to PyData Gandhinagar, could you add that file for me?

File: CONTRIBUTING.md (hunk 1 above)
Comment (kumiDa) [reply CONTRIBUTING.md]: Sure...I will change this.

File: CONTRIBUTING.md (hunk 2 above)
Comment (kumiDa) [reply CONTRIBUTING.md:20]: How exactly do you see this Contributors' Hall of fame to be implemented, auto-population of the file or manual filling of this file?
Can you give more specific ideas for this plan.

Comment (kumiDa) [reply CONTRIBUTING.md:20]: Yeah, will change the link to be specific to the repo.

Comment (sahitpj) [reply CONTRIBUTING.md:20]: I already finished working on that. Made a basic contributors.md file. Feel free to make it better

File: CONTRIBUTING.md (hunk 3 above)
Comment (kumiDa) [reply CONTRIBUTING.md]: Sure, I will link the Organisation's project listing here.

File: CONTRIBUTING.md (hunk 2 above)
Comment (kumiDa) [reply CONTRIBUTING.md:20]: Oh cool.
Just checked it. Guess I will link this in the CONTRIBUTING file in place of assistance.

**Comment by kumiDa** [comment]
I have made changes to the commits as requested in the review.
@sahitpj please check them.

**Review by sahitpj — APPROVED** [review]

**Comment by sahitpj** [comment]
thanks a lot @rahul-kumi for this. Go ahead and add your name to the contributors.md list and make a PR 😄

**Pull request merged**